gulpfile.js

```diff
@@ -19,7 +19,6 @@ const sassLint = require('gulp-sass-lint');
 const source = require('vinyl-source-stream');
 const sourcemaps = require('gulp-sourcemaps');
 const through = require('through2');
-const tabzilla = require('mozilla-tabzilla');
```

Comment:
Are we not using **tabzilla** anymore? Do we need to remove it from package.json?